Assets/HoloToolkit/Utilities/Scripts/GLTF/Scripts/GLTFComponent.cs.meta

```diff
@@ -1,11 +1,11 @@
 fileFormatVersion: 2
-guid: 3628db10e32278e499da95887f1bc989
-timeCreated: 1488842155
+guid: 9087a95f22bcee64ebc45f6deec08714
```

Comment:
Will this meta file change break the scene references in other people's projects?

Reply:
This is an update from the source repo. Looks like they changed it there, so I'm leaving this edit here to match it.